src/pkg/count/count.go

```diff
@@ -106,3 +108,33 @@ func (b *Bus) TotalValues() map[string]int64 {
 
 	return b.Values()
 }
+
+// ---------------------------------------------------------------------------
+// context embedding
+// ---------------------------------------------------------------------------
+
+type countKey string
```

Comment:
`countBusKey` instead of `countKey`?

Reply:
Eh, it's unexported.  Could just be named `key`.  But I don't think it's critical either way.